region/norfair/east/Spiky Acid Snakes Tunnel.json

```diff
@@ -169,14 +146,29 @@
       "link": [1, 2],
       "name": "Shinespark",
       "notable": false,
+      "entranceCondition": {
+        "comeInWithSpark": {}
+      },
+      "requires": [
+        "h_canNavigateHeatRooms",
+        {"shinespark": {"frames": 80, "excessFrames": 4}},
+        {"heatFrames": 210}
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "Come In Running and Shinespark",
+      "notable": false,
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 3,
+          "openEnd": 0
+        }
+      },
       "requires": [
         "h_canNavigateHeatRooms",
-        {"canComeInCharged": {
-          "fromNode": 1,
-          "framesRemaining": 0
-        }},
-        {"heatFrames": 200},
-        {"shinespark": {"frames": 76}}
+        {"shinespark": {"frames": 76, "excessFrames": 4}},
```

Comment:
this one is probably worth adding a midair option, since it can help a lot, and there are no frames to worry about.